src/rust/engine/remote_provider/remote_provider_reapi/Cargo.toml

```diff
@@ -21,6 +21,7 @@ tokio-util = { workspace = true, features = ["io"] }
 tonic = { workspace = true }
 uuid = { workspace = true, features = ["v4"] }
 workunit_store = { path = "../../workunit_store" }
+prost = { workspace = true }
```

Comment:
It looks like this should technically be in `dev-dependencies`, since it's only used by tests... but I suspect it doesn't matter given it's already in the non-dev dependency tree. Only worth moving if you're making other changes.